src/pages/Send/index.js

```diff
@@ -641,31 +643,31 @@ class Send extends Component {
       return (
         <div>
           <div>
-            You are selling {b(`${+inputValue} ${inputLabel}`)}.
+            {t("youAreSending")} {b(`${+inputValue} ${inputLabel}`)}.
```

Comment:
I originally thought this was supposed to be t("youAreSelling" i.e. selling vs sending)
However, I upon closer inspection I see that the account is actually "sending" (to the recipient account) so this is technically correct (for the Send) page.

Reply:
Yes, I did that on purpose, hope it's right.